src/content/reference/apis/request.md

```diff
@@ -63,6 +63,7 @@ Special information from an incoming request to help with your app's logic. All
 - `latitude`: Latitude of the incoming request. (e.g. `"30.27130"`)
 - `longitude`: Longitude of the incoming request. (e.g. `"-97.74260"`)
 - `postalCode`: PostalCode of the incoming request. (e.g. `"78701"`)
+- `metroCode`: MetroCode of the incoming request. (e.g. `"635"`)
```

Comment:
Would be good to also mention DMA (i.e. `Metro code (DMA) region`....

Reply:
I mentioned DMA but I didn't quite understand the "region" part. Happy to try again if it's still not quite right.